ray-operator/config/crd/bases/ray.io_rayservices.yaml

```diff
@@ -11471,85 +11471,73 @@ spec:
                 required:
                 - headGroupSpec
                 type: object
-              serveConfigs:
+              serveDeploymentGraphConfig:
                 description: 'Important: Run "make" to regenerate code after modifying
                   this file'
-                items:
-                  description: ServeConfigSpec defines the desired state of RayService
-                    Reference to https://docs.ray.
-                  properties:
-                    autoscalingConfig:
-                      additionalProperties:
-                        type: string
-                      type: object
-                    gracefulShutdownTimeoutS:
-                      format: int32
-                      type: integer
-                    gracefulShutdownWaitLoopS:
-                      format: int32
-                      type: integer
-                    healthCheckPeriodS:
-                      format: int32
-                      type: integer
-                    healthCheckTimeoutS:
-                      format: int32
-                      type: integer
-                    importPath:
-                      type: string
-                    initArgs:
-                      items:
-                        type: string
-                      type: array
-                    initKwargs:
-                      additionalProperties:
-                        type: string
-                      type: object
-                    maxConcurrentQueries:
-                      format: int32
-                      type: integer
-                    name:
-                      type: string
-                    numReplicas:
-                      format: int32
-                      type: integer
-                    rayActorOptions:
-                      description: RayActorOptionSpec defines the desired state of
-                        RayActor
+                properties:
+                  importPath:
+                    type: string
+                  runtimeEnv:
+                    type: string
+                  serveConfigs:
+                    items:
+                      description: ServeConfigSpec defines the desired state of RayService
+                        Reference to https://docs.ray.
```

Comment:
I don't think `https://docs.ray` is a live link. For now, could we use `http://rayserve.org`? We can replace it with a link to the K8s documentation once that's finished.